src/code/main.c

```diff
@@ -1,32 +1,40 @@
-/**
- * This file has unmigrated bss. It is not practical to migrate it until we have a better way of dealing with bss
- * reordering than just prevent_bss_reordering.h: there is too much of it to control, and it cannot be split into
- * separate files since most of it is at addresses ending in 8.
- */
-
-#include "global.h"
-#include "buffers.h"
+#include "idle.h"
+#include "irqmgr.h"
+#include "padmgr.h"
+#include "scheduler.h"
 #include "stack.h"
 #include "stackcheck.h"
 #include "system_heap.h"
+#include "z64.h" // TODO: remove when AudioMgr has been split out
+
+// Variables are put before most headers as a hacky way to bypass bss reordering
+OSMesgQueue sSerialEventQueue;
+OSMesg sSerialMsgBuf[1];
+u32 gSegments[NUM_SEGMENTS];
+SchedContext gSchedContext;
+IrqMgrClient sIrqClient;
+OSMesgQueue sIrqMgrMsgQueue;
+OSMesg sIrqMgrMsgBuf[60];
+OSThread gGraphThread;
+STACK(sGraphStack, 0x1800);
+STACK(sSchedStack, 0x600);
+STACK(sAudioStack, 0x800);
+STACK(sPadMgrStack, 0x500);
+StackEntry sGraphStackInfo;
+StackEntry sSchedStackInfo;
+StackEntry sAudioStackInfo;
+StackEntry sPadMgrStackInfo;
+AudioMgr sAudioMgr;
+static s32 pad;
```

Comment:
There are a lot of unused bss variables. Are we always going to name it `static s32 pad;`. I feel it could at least be `sBssPad` or something?

Reply:
`sBssPad` seems fine for me. I found a few other instances of bss variables just acting as pads that were called `sUnused`. I preferred `sBssPad` so changed them to match (as well as made them all static).

Reply:
At a quick glance, there's also:
`s32 unused; // Needed for bss` in `z_arrow_ice.c`
`static s32 unused = 0;` in `z_bg_iknin_susceil.c`